### PR TITLE
set MTG275-ZB-RL as whitelabel

### PR DESCRIPTION
**Pull request by Chartreusito**
I opened _TZE204_dtzziy1e to confirm it's a 24G sensor as advertised.
Works out of the box with the existing converter, just changed it as whitelabel to fix model and description.

### CODE REVIEW

File: src/devices/tuya.ts [hunk 1]
```diff
@@ -6054,10 +6054,13 @@ const definitions: Definition[] = [
         },
     },
     {
-        fingerprint: tuya.fingerprint('TS0601', ['_TZE204_sbyx0lm6', '_TZE204_dtzziy1e', '_TZE204_clrdrnya']),
+        fingerprint: tuya.fingerprint('TS0601', ['_TZE204_sbyx0lm6', '_TZE204_clrdrnya']),
```
Comment (Koenkk) [inline src/devices/tuya.ts:6057]: `_TZE204_dtzziy1e` shouldn't be removed here as the device will not be detected as supported anymore.

Comment (Chartreusito) [reply src/devices/tuya.ts]: Dully noted, thanks !

**Comment by Koenkk** [comment]
Thanks, could you also submit a pr for the picture?

**Comment by Chartreusito** [comment]
Is it necessary if the device is visually identical to the main device ?
Its looks the same as this : https://www.zigbee2mqtt.io/devices/MTG075-ZB-RL.html#tuya-mtg075-zb-rl
Internally it's a different sensor

**Comment by Koenkk** [comment]
Yes, otherwise the frontend won't show a picture for it.

**Comment by Chartreusito** [comment]
Picture PR for MTG275-ZB-RL : https://github.com/Koenkk/zigbee2mqtt.io/pull/2255

**Comment by striker54** [comment]
Soory in the first place if this isn't the right place.
When trying to adjust Illuminance threshold value it gives the following error **No converter available for 'illuminance_threshold'**
Anyone has the same problem?

**Comment by djcrawleravp** [comment]
Anyone having the same "spammy network" problem with the MTG275-ZB-RL?

**Comment by yfaykya** [comment]
> Anyone having the same "spammy network" problem with the MTG275-ZB-RL?

Did you ever find a solution?

**Pull request merged**